### ISSUE TITLE
UI Issues on First Launch

### ISSUE THREAD

**Issue by Kazuto88** [fifty-six/Scarab#102]
### Scarab Version

v1.19.0.0

### Hollow Knight Version

v1.5.78.11833

### Steps to Reproduce

Run Scarab for the first time.

### Expected Behaviour

The pop-up for detecting the game's path should be fully readable. Also, when Scarab loses focus, this window should be at the front when Scarab is given focus again.

### Actual Behaviour

The pop-up message for the game path gets cut off. See screenshot below. If the Scarab window loses focus, when you re-focus on the Scarab window, the pop-up gets hidden behind the main Scarab window, which is an entirely blank window at this point. See the second screenshot.

![image](https://user-images.githubusercontent.com/39844884/182669672-7de8c713-f39c-4934-9a66-f88c3cf3f178.png)

![image](https://user-images.githubusercontent.com/39844884/182670348-ab649719-7c36-429d-9726-e99b869a6040.png)

### Logs

None

**Comment by Kazuto88** [comment]
Forgot to mention, I'm running Windows 10 Pro x64 v21H2

**Comment by fifty-six** [comment]
Yeah - the former bit is known, it's a bug in the message box library I'm using, not entirely sure how to go about fixing that for now besides rewriting the message box lmao. I didn't know about the latter though, that's a bit scuffed. Do you remember if you can access it by doing the hovering over the taskbar icon thing in Windows?

**Comment by kibin** [comment]
having the same issue in crossover bottle windows 7-10 x64, but all the time. will try in parallels

**Comment by Kazuto88** [comment]
Apologies for the late reply! If this is still an issue, then no, hovering the icon on the taskbar doesn't help either. If I recall, it'll _show_ the message box window correctly, but when you click the icon, it'll just pull up the black window like in the second screenshot.
